### ISSUE TITLE
Multiple new lines should be flagged and squashed to one

### ISSUE THREAD

**Issue by vanniktech** [shyiko/ktlint#275]
Given the following file:

```kotlin
class Foo {
  ...
}


```

it should be flagged since there are multiple new lines and it should be auto formatted to:

```kotlin
class Foo {
  ...
}

```

I've tried quickly to contribute this but given that two build system are in use and the tests are all cramped into two big test methods it does not make it easy to quckly fix something upstream.

**Comment by shyiko** [comment]
```sh
$ ktlint --version
0.27.0

$ cat -A Foo.kt 
class Foo {$
    //$
}$
$

$ ktlint Foo.kt
/tmp/Foo.kts:5:1: Needless blank line(s)

$ ktlint -F Foo.kt 

$ cat -A Foo.kt 
class Foo {$
    //$
}$
```

Looks good to me.  
Are you hitting https://github.com/shyiko/ktlint/issues/253#issuecomment-406456664 by any chance?

> ... but given that two build system are in use ...

Ignore Gradle build scripts (they are strictly for Google's convenience). All the development is done using Maven.

**Comment by vanniktech** [comment]
Hmm weird. Just tried it again and this time it did work. I'll report back in case I find a reproducible test case.
